ivy/data_classes/array/conversions.py

```diff
@@ -17,23 +17,42 @@
 # --------#
 
 
-def _to_native(x: Any, inplace: bool = False, to_ignore: tuple = ()) -> Any:
+def _to_native(x, inplace: bool = False, to_ignore: tuple = None):
     to_ignore = ivy.default(to_ignore, ())
     if isinstance(x, to_ignore):
         return x
     if isinstance(x, ivy.Array):
         return x.data
-    # to prevent the graph from breaking for the time being
-    elif type(x) is ivy.Shape:
+    elif isinstance(x, ivy.Shape):
         return x.shape
+    elif is_frontend_array(x):
+        return x.ivy_array.data
+    elif is_frontend_shape(x):
+        return x.ivy_shape.shape
     elif isinstance(x, ivy.Container):
         return x.cont_map(
-            lambda x_, _: _to_native(x_, inplace=inplace, to_ignore=to_ignore),
```

Comment:
Also, let's pass the `to_ignore` argument in the recursive call

Reply:
It 's already there right ?

Reply:
yep it is, thanks!